lib/msf/core/opt_base.rb

```diff
@@ -196,6 +197,10 @@ def invalid_value_length?(value)
     #
     # A optional regex to validate the option value
     #
+    attr_accessor :conditions
+    #
+    # The list of potencial conditions
+    #
     attr_accessor :regex
```

Comment:
Comments go above the option name in question.

```suggestion
    #
    # The list of potential conditions
    #
    attr_accessor :conditions
    #
    # A optional regex to validate the option value
    #
    attr_accessor :regex
```

Reply:
Done!